assets/api-swagger/swagger.json

```diff
@@ -1970,6 +1970,184 @@
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            }
+          },
+          {
+            "name": "since_updated",
+            "in": "query",
+            "required": false,
+            "description": "Get translation requests updated since this timestamp, in ISO 8601 format.",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            }
+          },
+          {
+            "name": "limit",
+            "in": "query",
+            "description": "The number of records to fetch. Maximum 50.",
+            "required": false,
+            "schema": {
+              "type": "integer",
+              "default": 10
+            }
+          },
+          {
+            "name": "offset",
+            "in": "query",
+            "description": "The index of the first record to fetch.",
+            "required": false,
+            "schema": {
+              "type": "integer",
+              "default": 0
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of translation requests.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/TranslationRequest"
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid request."
+          },
+          "403": {
+            "description": "API key is not authorised to access this resource."
+          },
+          "500": {
+            "description": "An error occurred."
+          }
+        }
+      }
+    },
+    "/translations/requests/{request_id}": {
+      "get": {
+        "tags": [
+          "Translation Requests"
+        ],
+        "summary": "Get translation request details.",
+        "parameters": [
+          {
+            "name": "request_id",
+            "in": "path",
+            "required": true,
+            "description": "The id of the translation request.",
+            "schema": {
+              "type": "integer"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Translation request details.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/TranslationRequestDetails"
+                }
+              }
+            }
+          },
+          "403": {
+            "description": "API key is not authorised to access this resource."
+          },
+          "404": {
+            "description": "Translation request not found."
+          },
+          "500": {
+            "description": "An error occurred."
+          }
+        }
+      }
+    },
+    "/translations/submit": {
```

Comment:
`submit` makes sense, but we'll have to test if changing the endpoint name in our service works first.